tests/src/FunctionalJavascript/CustomFieldSubmissionTest.php

```diff
@@ -179,7 +179,7 @@ public function testSubmitWebform() {
 
     $this->createCustomFields();
 
-    $this->drupalLogin($this->adminUser);
+    $this->drupalLogin($this->rootUser);
     $this->drupalGet(Url::fromRoute('entity.webform.civicrm', [
```

Comment:
I looked further and I still don't know which permission our `adminUser` was missing (and was therefore losing form_key while Editing an Element with Options) - but `rootUser` is 👍